.changeset/stale-maps-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+remap index components
```

Comment:
not sure what would be correct entry for change log but I needed it in order to publish rc version to test in cloud-ui

Reply:
Is this introducing any changes to the `dist` files? How do these changes impact our consumers?